### DISCUSSION TITLE
Thank you so much for this tool !

### DISCUSSION THREAD

**Discussion by vdelord** [Breakthrough/DVR-Scan#106] [General]
Got back from vacation to find my garage has been emptied !
I processed 7 nights of video after tweaking a bit the command line and used the roi feature to speed up the processing.
I caught them in this morning processed events showing their faces to the camera <3

For the record, I used it on windows against 250 mp4 videos of 1Go each in 1080p.
dvr-scan.exe -l 20s -tb 5s -tp 5s -i *.mp4 -d night_events -roi

You saved me from days of painful reviews :D

Thank you so much.

Vincent

**Comment by Breakthrough** [comment]
Sorry to hear about the stolen items.  I really appreciate your kind message, thank you for sharing the parameters that you used!
